### PR TITLE
Unigo66 beta pcb added, bug fixes for flashing

### PR DESCRIPTION
**Pull request by danielhklein**
## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* Closes #5117

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sirius/unigo66/unigo66.h [hunk 1]
```diff
@@ -120,4 +120,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     KE1,KC_NO,K1D,K1B,K06,K19,K05,K11,K10,K36,K37,KC_NO,    KC_NO,KE5,      K52,      KC_NO,KC_NO,KC_NO,KC_NO, KC_NO,KC_NO, \
     KE0,KE3,KE2,KC_NO,KC_NO,    K2C,    KC_NO,KC_NO,KC_NO,KE6,KC_NO,KC_NO,KE4,  K50,K51,K4F,  KC_NO,      KC_NO,KC_NO, KC_NO,KC_NO  \
 )
-#endif
+
+#define LAYOUT_BetaPCB( \
```
Comment (drashna) [inline keyboards/sirius/unigo66/unigo66.h]: I don't know the naming for it, but usually, everything after `LAYOUT` is lower case.  Specifically:

```suggestion
#define LAYOUT_beta_pcb( \
```

Comment (danielhklein) [reply keyboards/sirius/unigo66/unigo66.h]: fixed in both unigo66.h and info.json

**Comment by danielhklein** [comment]
> Could you remove the uppercase readme file?

Done @drashna

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**